tests/integrations/test-integration.php

```diff
@@ -44,6 +61,8 @@ public function test__calling_activate_twice_on_same_integration_does_not_activa
 
 		$integration->activate();
 		$integration->activate();
+
+		$this->assertFalse( $integration->is_active() );
```

Comment:
If I understand correctly, this will be `false` due to the second `load()` call failing?

Reply:
Its false due to second `activate` I have just added it to make sure that after calling `trigger_error` we don't continue with the execution of the function.

Reply:
Right. Thanks!